container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -95,6 +95,16 @@ CONCURRENCY="${CONCURRENCY:-}"
 TEMPESTCONF_ARGS=""
 TEMPEST_ARGS=""
 
+function catch_error_if_debug {
+    echo "File run_tempest.sh has run into an error!"
+    sleep infinity
```

Comment:
it seems like (am not entirely sure) that "infinity" is just a bash thing, if i run this in a shell, it throws an error "usage: sleep seconds"

Also, do we want to sleep right here at the beginning when the debug mode is enabled?

Reply:
I think infinity needs to be defined on the system where the `sleep infinity` command is running (maybe macos is missing it? - `sleep infinity` worked on all my machines ubuntu, rhel, fedora, ...). 

If it is an issue then I suggest let's switch to:
```
while true; do sleep 10000; done
```

Reply:
if it works on the targeted systems, i don't mind it .. the second question still stands

Reply:
right, i've missed that

the sleep for no error situation is added on line 344 if i understand that correctly